esp-hal/src/twai/mod.rs

```diff
@@ -37,20 +49,22 @@
 //!     None,
 //! );
 //!
-//! // Partially filter the incoming messages to reduce overhead of receiving undesired messages
+//! // Partially filter the incoming messages to reduce overhead of receiving
+//! // undesired messages
 //! const FILTER: twai::filter::SingleStandardFilter =
-//!     twai::filter::SingleStandardFilter::new(b"xxxxxxxxxx0", b"x", [b"xxxxxxxx", b"xxxxxxxx"]);
-//! can_config.set_filter(FILTER);
+//!     SingleStandardFilter::new(b"xxxxxxxxxx0", b"x", [b"xxxxxxxx",
+//! b"xxxxxxxx"]); can_config.set_filter(FILTER);
 //!
-//! // Start the peripheral. This locks the configuration settings of the peripheral
-//! // and puts it into operation mode, allowing packets to be sent and
-//! // received.
+//! // Start the peripheral. This locks the configuration settings of the
+//! // peripheral and puts it into operation mode, allowing packets to be sent
+//! // and received.
 //! let mut can = can_config.start();
 //!
 //! loop {
 //!     // Wait for a frame to be received.
 //!     let frame = block!(can.receive()).unwrap();
 //!
+//!     /*
```

Comment:
this "huge" block of commented out code doesn't really help I guess (and it looks weird in the rendered documentation)

we should remove or replace it

Reply:
I removed it, thanks.